.github/workflows/pytests.yml

```
@@ -0,0 +1,63 @@
+name: 'Server PyTests'
+description: 'Run server pytest'
```

actionlint (1.7.4)

unexpected key "description" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)